ansible/roles/operator-pipeline/templates/openshift/tasks/digest-pinning.yml

```diff
@@ -79,3 +80,23 @@ spec:
           echo "Manifests are pinned."
           echo -n "false" | tee $(results.dirty_flag.path)
         fi
+    - name: verify-related-images
+      image: "$(params.pipeline_image)"
+      workingDir: $(workspaces.source.path)
+      script: |
```

Comment:
Shouldn't this include a check for `params.enabled` like the previous steps have?

Reply:
Good question.  Now that I look at it, I think we only need to check `params.enabled` in the `pin-digest` step.  The `verify-pinning` and the `verify-related-images` steps should run regardless of `params.enabled`.  `params.enabled` == false doesn't necessarily mean `dirty_flag` == true anymore since we support manual pinning as well.  I'll update this to remove the `params.enabled` from the `verify-pinning` step.

Reply:
But will that work for a CI pipeline run where pinning is disabled? We've found that it's necessary to propogate skipping behavior across all steps of a task if other tasks will depend on it from a pipeline.

Reply:
You're right.  I added the `params.enabled` checks for `verify-pinning` and `verify-related-images`.  Thanks for the review.